docs/docs/advanced/debugging.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 10
+title: 🐛 Debugging
+---
+
+In some cases, you'll want to debug your Dart Frog app, and you might have noticed that by adding just the breakpoint, the debugger doesn't stop there like it does by default in a Flutter app.
+Below, there are two quick and easy options for debugging code in Dart Frog.
+
+## Debugging with Dart Frog IDE Extension 🐸
+
+:::caution
+This extension is only available for Visual Studio Code, if you are using another IDE, please refer to the **Debugging by Attaching to Dart Process** section.
+:::
+
+:::info
+If you are interested in an extension for Android Studio or OpenVSX there are some open issues you can follow to learn more about it.
+
+- **Android Studio** [#1326](https://github.com/VeryGoodOpenSource/dart_frog/issues/1326)
+
+- **OpenVSX** [#907](https://github.com/VeryGoodOpenSource/dart_frog/issues/907)
+
+:::
+
+- Go to the Visual Studio Code Marketplace in [here](https://marketplace.visualstudio.com/items?itemName=VeryGoodVentures.dart-frog) and install the extension.
+
+:::info
+You can also install the extension by searching for `Dart Frog` in the extensions tab in Visual Studio Code.
+:::
+
+- Open your Dart Frog app.
+- Open **Visual Studio Command Palette** by clicking `Shift` + `Command` + `P` (Mac) / `Ctrl` + `Shift` + `P` (Windows/Linux).
+  You will see different options as shown in the image below.
+
+![Dart Frog Extension Options](../../static/img/dart_frog_extension_options.png)
+
+- Click on the `Dart Frog: Start and Debug Development Server` option. This will start the Dart Frog server in debug mode.
+
+:::info
+You can also select the `Dart Frog: Debug Development Server` option if you already have the server running.
```

Comment:
The Dart Frog server can also be run through "Code Lens", the small text that appears on top of the route function name. Or by using the Dart Frog icon within the Status Bar (appears you have a Dart Frog application open as in your IDE). 

Not sure if we could mention those here too.

Reply:
I added the code lens option, but in the status bar I only see the option to run the server, but it doesn't run in debug mode